seatunnel-core/seatunnel-starter/src/main/bin/seatunnel-cluster.sh

```diff
@@ -84,8 +84,12 @@ if [ -e "${CONF_DIR}/log4j2.properties" ]; then
   JAVA_OPTS="${JAVA_OPTS} -Dseatunnel.logs.file_name=seatunnel-engine-server"
 fi
 
-echo "JAVA_OPTS: ${JAVA_OPTS}"
-
 CLASS_PATH=${APP_DIR}/lib/*:${APP_JAR}
 
+ST_TMPDIR=`java -cp ${CLASS_PATH} org.apache.seatunnel.core.starter.seatunnel.jvm.TempDirectory`
```

Comment:
Do We need to write a class to generate a temporary directory? Why not using shell instead of this? cc @EricJoy2048 @Hisoka-X

Reply:
I'm not sure the temp dir use for? Can you give me some tips?

Reply:
Is the latter engine considered for deployment in a windows environment?
If the future expectation is compatible with windows need this class, so that the logic of .sh and .bat files will be the same

Reply:
@MonsterChenzhuo Can you provide `JAVA_OPTS` after add `ST_TMPDIR`? Thanks

Reply:
cc @EricJoy2048